src/ui/router/Browse.js

```diff
@@ -119,12 +119,15 @@ define([
         function navigateToFirstChildOfRoot() {
             openmct.objects.get('ROOT')
                 .then(rootObject => {
-                    openmct.composition.get(rootObject).load()
+                    const composition = openmct.composition.get(rootObject);
+                    if (!composition) {
+                        return;
+                    }
+
+                    composition.load()
                         .then(children => {
                             let lastChild = children[children.length - 1];
-                            if (!lastChild) {
-                                console.error('Unable to navigate to anything. No root objects found.');
```

Comment:
I think this might be useful for developers to know why Open MCT is showing nothing. I agree that it generates a ton of noise in the logs though. Maybe as a compromise we could use `console.debug()` for the message, and set our minimum logging level in tests to be `info`?